examples/exec-healthz/README.md

```diff
@@ -0,0 +1,12 @@
+### SimpleHealthzServer
+
+SimpleHealthzServer is a sidecar container meant to serve as an exec-over-http bridge.
```

Comment:
Do you expect the exec to happen synchronously to thet healthz GET or async?  I'm assuming it is async, in which case s/over/to/ in this line.  But if it is sync I will have more comments later.